### PR TITLE
Dialog 5.7 locator change

### PR DESCRIPTION
**Pull request by sshveta**
{{pytest: -v --long-running cfme/tests/services/test_dynamicdd_dialogelement.py}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 9183a424e93d5445de00957a7259966e34f05381:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: cfme/automate/explorer.py [hunk 1]
```diff
@@ -176,7 +176,9 @@ def copy_to(self, domain=None):
                     "There is only one domain to select and that is {}".format(domain_selected))
             fill(self.copy_form, {"override": True})
         sel.click(self.copy_button)
-        flash.assert_message_match("Copy selected Automate {} was saved".format(self.class_name))
+        if version.current_version() < "5.7":
+            flash.assert_message_match("Copy selected Automate {} was saved".
+                format(self.class_name))
```
Comment (psav) [inline cfme/automate/explorer.py]: So there is no flash message after 5.7?

File: cfme/automate/service_dialogs.py [hunk 2]
```diff
@@ -29,7 +29,8 @@
                     '5.4': "//div[@class='dhxcont_global_content_area']"
                            "[not(contains(@style, 'display: none'))]/div/div/div/div/div/div"
                            "/div/div/div/ul[@class='dynatree-container']",
-                    '5.5': "//div[@class='modal-content']/div/div/ul[@class='dynatree-container']"})
```
Comment (psav) [inline cfme/automate/service_dialogs.py]: As we're here, why not remove the 5.4 and set the 5.5 to version.LOWEST

File: cfme/automate/explorer.py (hunk 1 above)
Comment (sshveta) [inline cfme/automate/explorer.py]: Yes no flash message in 5.7

**Comment by dajoRH** [comment]
Lint report for commit 3274cd4dad5a484c91bdd4912ba17cfb197b8816:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit c9c4f92052301212491f3820cc8d062a98e4c75d:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: cfme/automate/service_dialogs.py (hunk 2 above)
Comment (sshveta) [inline cfme/automate/service_dialogs.py]: Done

**Comment by dajoRH** [comment]
Lint report for commit e48691ada2dd7e8692e1b2d33307cd53ce8aaede:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Pull request merged**